<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XY_Widget</class>
 <widget class="QWidget" name="XY_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="Y_UP">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-17000</number>
   </property>
   <property name="maximum">
    <number>17000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="X_Left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>-50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="X_Right">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>48</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="Y_Down">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-17000</number>
   </property>
   <property name="maximum">
    <number>17000</number>
   </property>
   <property name="value">
    <number>-1000</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>520</width>
     <height>520</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>520</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>520</width>
     <height>520</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QSpinBox" name="Average_spinBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
